othdrplot/process_high_density_region_algorithm.py

```diff
@@ -157,44 +158,58 @@ def plotTrajectories(self):
             graph.add(traj_curves)
 
         # Plot mean
-        mean_field = self.processSample.computeMean()
-        mean_curve = ot.Curve(t, mean_field.getValues(), 'Discrete mean')
+        if discreteMean:
```

Comment:
The code is fine, but I would rather call it "central_field", given that in the case that this is the "mode", the name "mean_field" might be misleading. It does not prevent from pushing the code anyway.